### ISSUE TITLE
Feature Request : Tableau TDE hook and Tableau TDE create/refresh operator

### ISSUE THREAD

**Issue by syvineckruyk** [apache/airflow#196]
Using the Tableau Data Extraction API this should be possible.

Caveat 1 - @mistercrunch  curious to get your thoughts on this ... I don't believe the code from Tableau is open source so would need to review the license or not include their code in the package (up to user to install outside of Airflow). 

Caveat 2 - Code runs on Windows and Linux only ... which should be fine but would need to be called out as a constraint ...

Thinking this may be useful as a plugin rather than integration into Airflow considering the above caveats.

**Comment by mistercrunch** [comment]
We've been doing different things around Tableau integration but I don't think we've find something we're happy with. I wrote a little framework that would scan a Hive database ("tableau") and discover the all the tables from the metastore and push CSV files using the HiveToSamba operator. I think it would do differential loads where tables were partitionned.

I think we ended up settling on a pull approach using the Hortonworks hiveserver2 connector. 

We're interested in better integration between Tableau and Hive/Presto though. I haven't looked at the Tableau APIs lately.
